### ISSUE TITLE
[bug] setTimeout(...).unref is not a function

### ISSUE THREAD

**Issue by yafp** [yafp/media-dupes#123]
This happens in the latest dev version when adding an url to the todo-list - most likely related with metascraper and/or **got**.

```
RequestError: setTimeout(...).unref is not a function
    at ClientRequest.<anonymous> (/home/fidel/Dropbox/Temp/media-dupes/node_modules/got/dist/source/core/index.js:786:25)
    at Object.onceWrapper (events.js:300:26)
    at ClientRequest.emit (events.js:215:7)
    at ClientRequest.EventEmitter.emit (domain.js:476:20)
    at ClientRequest.origin.emit (/home/fidel/Dropbox/Temp/media-dupes/node_modules/@szmarczak/http-timer/dist/source/index.js:39:20)
    at TLSSocket.socketErrorListener (_http_client.js:406:9)
    at TLSSocket.emit (events.js:210:5)
    at TLSSocket.EventEmitter.emit (domain.js:476:20)
    at emitErrorNT (internal/streams/destroy.js:92:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:60:3)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)
    at CacheableLookup.tick (/home/fidel/Dropbox/Temp/media-dupes/node_modules/cacheable-lookup/source/index.js:290:22)
    at CacheableLookup.query (/home/fidel/Dropbox/Temp/media-dupes/node_modules/cacheable-lookup/source/index.js:176:8)
    at CacheableLookup.lookupAsync (/home/fidel/Dropbox/Temp/media-dupes/node_modules/cacheable-lookup/source/index.js:137:27)
    at CacheableLookup.lookup (/home/fidel/Dropbox/Temp/media-dupes/node_modules/cacheable-lookup/source/index.js:121:8)
    at net.js:1030:5
    at defaultTriggerAsyncIdScope (internal/async_hooks.js:301:12)
    at lookupAndConnect (net.js:1029:3)
    at TLSSocket.Socket.connect (net.js:963:5)
    at Object.connect (_tls_wrap.js:1434:13)
    at Agent.createConnection (https.js:125:22)
    at Agent.createSocket (_http_agent.js:234:26)
    at Agent.addRequest (_http_agent.js:193:10)
    at new ClientRequest (_http_client.js:276:16)
    at request (https.js:309:10)
    at PromisableRequest._makeRequest (/home/fidel/Dropbox/Temp/media-dupes/node_modules/got/dist/source/core/index.js:904:39)
    at /home/fidel/Dropbox/Temp/media-dupes/node_modules/got/dist/source/core/index.js:265:28
```

---

media-dupes 0.8.20200428
Electron 8.2.4
linux 5.6.7-arch1-1
Locale: de

**Comment by yafp** [comment]
https://github.com/microlinkhq/metascraper/issues/267

**Comment by yafp** [comment]
seems like using got 11.0.3 is causing issues while 10.7.0 works great.
the related metascraper version seems not to change the situation.

**Comment by yafp** [comment]
seems to be fixed with got 11.1.1
